vault/seal/ocikms/ocikms.go

```diff
@@ -16,30 +16,33 @@ import (
 	"math"
 	"os"
 	"strconv"
+	"sync/atomic"
 	"time"
 )
 
 const (
 	// OCI KMS key ID to use for encryption and decryption
 	EnvOCIKMSSealKeyID = "VAULT_OCIKMS_SEAL_KEY_ID"
-	// OCI KMS key ID to use for encryption and decryption
+	// OCI KMS crypto endpoint to use for encryption and decryption
 	EnvOCIKMSCryptoEndpoint = "VAULT_OCIKMS_CRYPTO_ENDPOINT"
+	// OCI KMS management endpoint to manage keys
+	EnvOCIKMSManagementEndpoint = "VAULT_OCIKMS_MANAGEMENT_ENDPOINT"
 	// Maximum number of retries
 	KMSMaximumNumberOfRetries = 5
 	// keyID config
 	KMSConfigKeyID = "keyID"
 	// cryptoEndpoint config
 	KMSConfigCryptoEndpoint = "cryptoEndpoint"
+	// managementEndpoint config
+	KMSConfigManagementEndpoint = "managementEndpoint"
 	// authTypeAPIKey config
 	KMSConfigAuthTypeAPIKey = "authTypeAPIKey"
 )
 
 const (
-	// OCIKMSEncrypt is used to directly encrypt the data with KMS
-	OCIKMSEncrypt = iota
 	// OCIKMSEnvelopeAESGCMEncrypt is when a data encryption key is generated and
 	// the data is encrypted with AESGCM and the key is encrypted with KMS
-	OCIKMSEnvelopeAESGCMEncrypt
+	OCIKMSEnvelopeAESGCMEncrypt = iota
```

Comment:
You can remove this since you do not have multiple versions for your encryption algorithm.